docs/BR.md

```diff
@@ -1532,24 +1493,22 @@ The CA MAY support revocation of Short-lived Subscriber Certificates.
 With the exception of Short-lived Subscriber Certificates, the CA SHALL revoke a Certificate within 24 hours and use the corresponding CRLReason (see Section 7.2.2) if one or more of the following occurs:
 
 1. The Subscriber requests in writing, without specifying a CRLreason, that the CA revoke the Certificate (CRLReason "unspecified (0)" which results in no reasonCode extension being provided in the CRL);
-2. The Subscriber notifies the CA that the original certificate request was not authorized and does not retroactively grant authorization (CRLReason #9, privilegeWithdrawn);
```

Comment:
@karolina-ads can you please check if something was removed by mistake? I don't think removal of these revocation cases should be part of a clean-up ballot.